psbt/Cargo.toml

```diff
@@ -25,6 +25,7 @@ bitcoin_hd = { version = "0.8.2", path = "../hd" }
 bitcoin_onchain = { version = "0.8.0", path = "../onchain" }
 serde_crate = { package = "serde", version = "1", optional = true }
 serde_with = { version = "1.12", optional = true }
+colored = { version = "2" }
```

Comment:
We clearly do not need to add a dependency here which would not be used by the downstream. If you need it just for the command-line tool than it has to be feature-gated

Reply:
Sure

Reply:
Ok